mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -726,3 +760,48 @@ func testAccMongoDBAtlasClusterAWSConfigdWithLabels(projectID, name, backupEnabl
 		}
 	`, projectID, name, backupEnabled, tier, region, labelsConf)
 }
+
+func testAccMongoDBAtlasClusterConfigWithPrivateEndpointLink(awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
+	return fmt.Sprintf(`
+		provider "aws" {
+			region     = "us-east-1"
```

Comment:
should this be hardcoded? or can we just take `region` from the param and make it dynamic

Reply:
I agree with you, make the region dynamic is better 👍

Reply:
you're right, already pushed changes 👍

Reply:
I think we should also make `region` dynamic in addition to the cluster name?